doc/adrs/122_add_support_for_board_representation.adoc

```diff
@@ -0,0 +1,110 @@
+This new type of representation can be seen as a kind of dashboard.
+It will be possible to add existing representations only.
+The user must be able to arrange the layout of these representations as he wishes.
+This representation can be created anywhere in the model tree.
+
+To date, the sub-representations available to populate a _board_ are:
+
+* diagram
+* form
+* gantt
+* deck
+* other board
+
+== Solution
+
+On any semantic element of the explorer, a user can create a new _board_ representation via the existing _New Representation_ toolbar menu.
+
+image:images/122/122_board_representation_creation.png[board_representation_creation]
+
+=== Board representation creation
+
+In this case, the main part of the page is empty.
+
+image:images/122/122_empty_main_view.png[empty_main_view]
+
+User can drag and drop existing representations from the explorer.
+When the first representation has been added to the _board_, UI switch to the edit mode.
+
+=== Edit mode
+
+* In this mode, it's possible to change the position and the size of all the representations.
+New representations can also be added by dragging them from the explorer.
+* The representations can be reorganized by drag and drop action.
+All the representations can be resized via a handle placed in the bottom right corner of the representation.
+If the resize of a representation overlaps another one, this second representation is moved away.
+* A board-type representation, has no horizontal or vertical boundaries.
+In case of overflow, there will be scrollbars for navigating.
+If a user doesn't want the page to be scrolled, it is possible by adjusting the layout.
+* Only in this mode, a representation can be deleted by using the _del_ key.
+This deletion does not affect the layout of the other representations.
+* When editing the _board_, the representations are read-only.
+* Since a board representation is itself a representation, it can be added to another _board_.
+There is limitation to avoid a _board_ including another _board_ being added to this second _board_.
+It prevents a _board_ loop.
+
+=== Direct mode
+
+* In this mode, the representations can't be moved or resized.
+* Representations on a _board_ have exactly the same behavior as if they were opened individually.
+A user can thus modify them.
+* If the representation provides one, the user has access to its palette.
+
+image:images/122/122_board_representation_example.png[board_representation_example]
+
+* If the defined size of a representation does not allow it to be displayed in its entirety, a scroll bar will be added.
+For diagram-type representations, it is possible to zoom inside.
+* Selecting an element from the explorer or from one of the _board_ representations changes the selection in all representations.
+* If a representation is deleted but always referenced in a _board_, its location is replaced by a blank slot with a message indication that the representation doesn't exist
```

Comment:
So if a representation is included in 10 boards, I have to update every boards manually? It seems cumbersome.